test/test.js

```diff
@@ -1,135 +1,57 @@
-'use strict';
-var fs = require('fs');
-var spawn = require('child_process').spawn;
-var test = require('ava');
-var pathExists = require('path-exists');
+import fs from 'fs';
+import {spawn} from 'child_process';
+import test from 'ava';
+import pathExists from 'path-exists';
+import execFile from 'get-exec-file';
+import {version as pkgVersion} from '../package.json';
 
 process.chdir(__dirname);
 
-test('generate screenshot', function (t) {
-	t.plan(2);
+test.serial('generate screenshot', async t => {
+	await execFile('../cli.js', ['yeoman.io', '320x240']);
 
-	var cp = spawn('../cli.js', ['yeoman.io', '320x240'], {
-		stdio: [process.stdin, null, null]
-	});
-
-	cp.on('close', function () {
-		t.assert(pathExists.sync('yeoman.io-320x240.png'));
-
-		fs.unlink('yeoman.io-320x240.png', function (err) {
-			t.assert(!err, err);
-		});
-	});
-});
-
-test('generate screenshots from a list of screen resolutions', function (t) {
-	t.plan(4);
-
-	var read = fs.createReadStream('fixture.txt');
-	var cp = spawn('../cli.js', ['yeoman.io']);
-
-	cp.on('close', function () {
-		t.assert(pathExists.sync('yeoman.io-1440x900.png'));
-		t.assert(pathExists.sync('yeoman.io-1280x1024.png'));
-		t.assert(pathExists.sync('yeoman.io-768x1024.png'));
-		fs.unlinkSync('yeoman.io-1440x900.png');
-		fs.unlinkSync('yeoman.io-1280x1024.png');
-
-		fs.unlink('yeoman.io-768x1024.png', function (err) {
-			t.assert(!err, err);
-		});
-	});
-
-	read.pipe(cp.stdin);
+	t.true(pathExists.sync('yeoman.io-320x240.png'));
+	fs.unlinkSync('yeoman.io-320x240.png');
 });
 
-test('remove temporary files on cancel', function (t) {
-	t.plan(2);
+test('remove temporary files on cancel', t => {
+	t.plan(1);
 
-	var cp = spawn('../cli.js', ['yeoman.io', '320x240'], {
-		stdio: [process.stdin, null, null]
-	});
+	const cp = spawn('../cli.js', ['yeoman.io', '320x240']);
 
-	cp.on('exit', function () {
-		fs.readdir(__dirname, function (err, files) {
-			t.assert(!err, err);
-			t.assert(files.indexOf('yeoman.io-320x240.png') === -1);
-		});
-	});
+	cp.on('exit', () => t.false(pathExists.sync('yeoman.io-320x240.png')));
 
-	setTimeout(function () {
+	setTimeout(() => {
 		cp.kill('SIGINT');
 	}, 500);
 });
 
-test('show error if no url is specified', function (t) {
-	t.plan(1);
-
-	var cp = spawn('../cli.js', ['320x240'], {
-		stdio: [process.stdin, null, null]
-	});
-
-	cp.stderr.setEncoding('utf8');
-	cp.stderr.on('data', function (data) {
-		t.assert(/Specify a url/.test(data), data);
-	});
+test('show error if no url is specified', async t => {
+	t.throws(execFile('../cli.js', ['320x240']), /Specify a url/);
```

Comment:
Weird, some guy already submitted a similar bug to AVA and I pushed a PR to fix it. @SamVerschueren could you please try to create a reproducible test case with only the basics, so that I can fix it?